includes/wf_crm_webform_postprocess.inc

```diff
@@ -1301,6 +1301,30 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
         // Update mode
         else {
           $params['id'] = $this->ent['activity'][$n]['id'];
+
+          // Update details when user has selected he wants to update the details
+          if (empty($this->data['activity'][$n]['details']['no_update_existing'])) {
```

Comment:
Because you've implemented this as a negative (do not update), this is going to change the behavior of all existing installs, because this condition will be true for all of them. Doing it as a positive (update details) would keep existing forms the same.